src/static/stilyagi/assets/styles/colors-and-type.css

```
@@ -0,0 +1,297 @@
+  --accent: var(--press-red);
+  /* Red type, resolved against whatever ground it sits on.  Dark panels
+     re-point this below, and because custom properties inherit, every
+     descendant — including elements carrying an inline colour — follows. */
+  --accent-text: var(--press-red-text);
+  --border: var(--ink);
+  --border-soft: rgba(15, 15, 15, 0.25);
+  --selection-bg: var(--press-red);
+  --selection-fg: var(--paper);
+
+  /* ---------- TYPE ------------------------------------------------- */
+  --font-display: "Big Shoulders Display", Impact, "Arial Narrow", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Impact" to be "impact" (value-keyword-case)

(value-keyword-case)